app/models/post.rb

```diff
@@ -1,2 +1,16 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_post_counter
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
+  end
+
+  private
```

Comment:
- [ ] Kindly make sure the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).👍
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```